src/sql/workbench/contrib/extensions/browser/extensionsActions.ts

```diff
@@ -85,3 +87,58 @@ export class OpenExtensionAuthoringDocsAction extends Action {
 		await this.openerService.open(URI.parse(OpenExtensionAuthoringDocsAction.extensionAuthoringDocsURI));
 	}
 }
+
+export class HidePanel extends Action {
+	static readonly ID = 'workbench.extensions.action.hidePanel';
+	static readonly LABEL = localize('hidePanel', "Hide the panel...");
+
+	constructor(
+		id: string = HidePanel.ID,
+		label: string = HidePanel.LABEL,
+		@IWorkbenchLayoutService private readonly layoutService: IWorkbenchLayoutService
+	) {
+		super(id, label);
+	}
+
+	override async run(): Promise<void> {
+		this.layoutService.setPanelHidden(true);
+	}
+}
+
+export class HideSettings extends Action {
+	static readonly ID = 'workbench.extensions.action.hideSettings';
+	static readonly LABEL = localize('hideSettings', "Hide the settings icon...");
+
+	constructor(
+		id: string = HideSettings.ID,
+		label: string = HideSettings.LABEL,
+	) {
+		super(id, label);
+	}
+
+	override async run(): Promise<void> {
+		let allActionItems = Array.from(document.getElementsByClassName('action-item icon'));
+		let manageElement = allActionItems.filter((el) => el.getAttribute('aria-label') === 'Manage');
+		manageElement[0].parentNode.removeChild(manageElement[0]);
+	}
+}
+
+export class HideActivityBarViewContainers extends Action {
+	static readonly ID = 'workbench.extensions.action.hideActivityBarViewContainers';
+	static readonly LABEL = localize('hideActivityBarViewContainers', "Hide the extension viewlet...");
+
+	constructor(
+		id: string = HideActivityBarViewContainers.ID,
+		label: string = HideActivityBarViewContainers.LABEL,
+		@IActivityBarService private readonly activityBarService: IActivityBarService,
+	) {
+		super(id, label);
+	}
+
+	override async run(): Promise<void> {
+		let array = ['workbench.view.search', 'workbench.view.explorer', 'workbench.view.scm', 'workbench.view.extensions'];
```

Comment:
This is the same code as the contribution you made above (you just are never using the parameter you pass it in the contribution).

You should either make this only hide one container or only call it once in the contribution.